src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -371,25 +371,28 @@ var ReactCompositeComponentMixin = {
    * @private
    */
   _processChildContext: function(currentContext) {
+    var Component = ReactNativeComponent.getComponentClassForElement(
+      this._currentElement
+    );
```

Comment:
This should not be necessary because the wrappers are getting removed in another PR. Also, no wrapper should need to use child context.